src/shared/utils/nexus-maybe.ts

```diff
@@ -3,8 +3,9 @@ import { Resource } from '@bbp/nexus-sdk-legacy';
 const NEXUS_FILE_TYPE = 'File';
 const VIEW = 'View';
 const ES_VIEW = 'ElasticSearchView';
+const AGGREGATE_ES_VIEW = 'AggregateElasticSearchView';
+const AGGREGATE_SPARQL_VIEW = 'AggregateSparqlView';
 const SPARQL_VIEW = 'SparqlView';
-
 const DEFAULT_ELASTIC_SEARCH_VIEW_ID = 'nxv:defaultElasticSearchIndex';
 
 const isOfType = (type: string) => (resource: Resource) =>
```

Comment:
Why is it still using `Resource` from the legacy sdk?

Reply:
it was made before the conversion to v1.1. Should it be any object, or a new ResourceResponseCommon or so?

Reply:
Yes, either you create a new type or you use the new SDK one. But if you add new functionality or update something in general, make sure you refactor and remove the legacy dependencies.

Reply:
Also the views that is using this have the resources in the format such as resource[`@type`] is actually resource.type, so without changing that up the chain for the Resource view we'll have problems using this.